server/daemon/src/main.rs

```diff
@@ -145,6 +147,22 @@ async fn submit_admin_req(path: &str, req: AdminTaskRequest, output_mode: Consol
                 info!(new_password = ?password)
             }
         },
+        Some(Ok(AdminTaskResponse::ShowReplicationCertificate { cert })) => match output_mode {
+            ConsoleOutputMode::JSON => {
+                eprintln!("{{\"certificate\":\"{}\"}}", cert)
+            }
+            ConsoleOutputMode::Text => {
+                info!(certificate = ?cert)
+            }
+        },
+        Some(Ok(AdminTaskResponse::RenewReplicationCertificate { success })) => match output_mode {
```

Comment:
We really should show the cert here 'cause folks are going to be pulling it next anyhow.

Reply:
Yep, that's doable :)